jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -605,6 +606,61 @@ def vue_set_value(self, data):
         value = data.get('value')
         setattr(self, attr_name, value)
 
+    def apply_RGB_presets(self):
+        """
+        Applies preset colors, opacities, and stretch settings to all visible layers
+        (in all viewers) when in Monochromatic mode.
+        """
+
+        if self.image_color_mode_value != "One color per layer":
+            raise ValueError("RGB presets can only be applied if color mode is Monochromatic.")

```

Comment:
For "RGB" workflows specifically, this makes sense, but I wonder if there are uses for the rest of the settings (stretch, etc) even in colormap mode?  The other benefit to loosening this restriction would be that mixed-colormode would work out of the box.  The downside is we might want to consider naming it something more general and explain in the tooltip/docs what to expect when clicking the button.

Reply:
(If you want to leave this, that's ok with me, we just need to make sure the mixed case works as expected)

Reply:
I would prefer to leave as-is for now. I confirmed that if you have another viewer on `Colormaps` mode, it doesn't change those settings (or break) when you use `Apply Presets` on the viewer in `Monochromatic` mode. The button also does not appear when multiple viewers with different modes are selected with multi-select. Does that cover the case(s) you were thinking about?

Reply:
`image_color_mode_value` will be dependent on the _order_ in which the viewers were selected when the colormode in those selected viewers are mixed.  Maybe this is ok for now, or maybe we should just check to see if they're mixed as well here and in the `v-if`?  That information is stored in `image_color_mode_sync['mixed']`.


https://github.com/spacetelescope/jdaviz/assets/877591/745fb2a1-ac60-42ab-b1b5-d8b91628f902

Reply:
I think just adding `or image_color_mode_sync['mixed']` here and to the `v-if` is good enough for now and we can always add support for colormaps later.

Reply:
Implemented that, think I got it right.